src/chef-server-ctl/plugins/gather-logs.rb

```diff
@@ -2,11 +2,10 @@
 #
 # All Rights Reserved
 #
-
-add_command_under_category "gather-logs", "general", "Create a tarball of recent logs and system information for Chef Support", 2 do
+add_command_under_category "gather-logs", "general", "Create a tarball of recent logs and system information for #{ChefUtils::Dist::Server::PRODUCT} Support", 2 do
```

Comment:
Similar to one above, this should be appropriate replacement for Chef instead of replacing with Chef Infra Server, if possible.

Reply:
That makes sense. Now that I think about this... should we _not_ replace this word and actually intentionally leave it as `Chef Support`? Because, in this case it is actually talking about getting support from Progress/Chef, and if we were to replace this with E.G. "Cinc Support" (for the Cinc project) that doesn't make sense.

Up to you and @tas50, whatever you prefer here I'll do! 😄